ui/js/controllers/main.js

```diff
@@ -26,7 +26,7 @@ treeherderApp.controller('MainCtrl', [
             return "[" + ufc + "] " + $rootScope.repoName;
         };
 
-        $scope.closeJob = function() {
+        $rootScope.closeJob = function() {
```

Comment:
yeah, if it's ``$rootScope`` then it's accessible from any ``$scope`` as well.  I thought this change would be pretty ok since this function actually acts on lots of ``$rootScope`` stuff already.  It seemed to make sense.